jobs/nats-tls/spec

```diff
@@ -79,6 +79,10 @@ properties:
   nats.write_deadline:
     description: "Maximum number of seconds the server will block when writing. Once this threshold is exceeded the connection will be closed and the client will be considered as Slow Consumer."
     default: "2s"
+  nats.net:
+    decription: "Client lisenting interface, defaults to spec.address"
+  nats.cluster_host:
+    decription: "Clustering lisenting interface, defaults to spec.address"
```

Comment:
Can you fix the `listening` typos here?

Reply:
oops, yes... "decription" and "lisenting" have been corrected ('^_^) thank you for your review